app/src/renderer/components/common/PageProfile.vue

```diff
@@ -1,19 +1,18 @@
 <template lang="pug">
 page(title="Your Profile")
-  tool-bar
-    anchor-copy(:value="user.pubkey" icon="content_copy")
+  div(slot="menu"): tool-bar
+    anchor-copy(value="TODO: user.pubkey" icon="content_copy")
     a(@click.native='signOut') Sign Out
 
   part(title='Profile')
-    list-item(dt="Total Vote Power" :dd="user.atoms" to="/vote-power")
+    list-item(dt="Total Vote Power" dd="TODO: user.atoms" to="/vote-power")
     list-item(dt="Solo Vote Power" dd="31.2M ATOM (19%)")
     list-item(dt="Delg. Vote Power" dd="33.8M ATOM (81%)")
     list-item(dt="Vote History" dd="37 Votes" to="/votes")
     list-item(dt="Proposals" dd="13" to="/proposals")
```

Comment:
only link that "works"

Reply:
Just removed all the links that we don't have pages for yet.